docs/attributes-registry/k8s.md

```diff
@@ -15,8 +15,10 @@ Kubernetes resource attributes.
 
 | Attribute                                     | Type   | Description                                                                                                                                                      | Examples                                                                                                                                  | Stability                                                        |
 | --------------------------------------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `csi.driver`                                  | string | The name of the CSI driver used by the volume. [1]                                                                                                               | `pd.csi.storage.gke.io`                                                                                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Given CSI is a API specification that can be used by any container orchestrator and not only k8s, it's probably wrong that these attributes are part of the k8s file, no? 

I'm no expert, but to me it seems `csi` should be its own thing in the registry, and then `k8s` just refer to its attributes.

Reply:
I think that's right. I think what's not clear to me is how these attrs should be named generically for all of CSI.

Kube calls this particular field a "driver". CSI seems to refer to the overall interface as a "plugin". In the spec's protobuf this field is returned as part of GetPluginInfo, a `name` field. So should this be called `csi.plugin.name`?

For the other field, `csi.volume.handle`, this also is a k8s-named thing. A handle in this case seems to map to a CreateVolumeResponse, which returns a [Volume](https://github.com/container-storage-interface/spec/blob/2f340622302a8d26950aba0f73f240ea1132a67b/csi.proto#L552), which has `volume_id`. Maybe `csi.volume.id`?

I was a little uneasy making these assumptions without some extra domain knowledge across the other CSI implementations, so I mapped 1:1 with k8s, but maybe we can decide on something reasonable.

Reply:
Yeah I think we could go either way. But of course, the most feature proof way is that CSI is it's own "thing" in semconv, having attributes that map to their concepts (like you said, Plugins). And then each implementation like k8s just import them.

Reply:
I did the bits above and moved to a new component in https://github.com/open-telemetry/semantic-conventions/pull/1337/commits/c352b3de8eee816867a2c653fa4b3ab551f28f8c, and set the k8s attrs to refer to it as "recommended"...I'm not sure I did that right, but PTAL!